stage0/src/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ endif()
 include(ExternalProject)
 project(LEAN CXX C)
 set(LEAN_VERSION_MAJOR 4)
-set(LEAN_VERSION_MINOR 2)
+set(LEAN_VERSION_MINOR 3)
```

Comment:
Please rebase your changes onto master. We shouldn't be seeing diffs like this.

Reply:
I rebased the PR onto master. Perhaps there has been no stage0 update on master in a while?

Reply:
@semorrison I've now rebased this PR onto the recent stage0 update in master. The stage0 diff is a lot smaller.